woocommerce/woocommerce-admin
# Fix styling of chart component header.

Fixes #6863.

This PR seeks to fix the styling regressions introduced in https://github.com/woocommerce/woocommerce-admin/pull/5753. The root cause was an update to the `SelectControl` component in `@wordpress/components` that no longer wraps the outermost element with the `className` prop.

### Accessibility




### Screenshots

<img width="1133" alt="Screen Shot 2021-05-17 at 4 43 44 PM" src="https://user-images.githubusercontent.com/63922/118554035-4d607100-b71e-11eb-9876-7288774da5fa.png">

### Detailed test instructions:

- View a report (Analytics > Revenue)
- Verify the correct styling of the chart header
- Verify correct styling when tabbing to focus the chart interval selection
